### PR TITLE
Create new

### PR DESCRIPTION
**Pull request by AntoniaKro**

### CODE REVIEW

File: src/app/App.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
+import React, { useState } from 'react';
 import styled, { createGlobalStyle } from 'styled-components';
 import Header from '../components/Header/Header';
-import AdFeed from '../components/FeedPage/AdFeed.js';
+import ListingFeed from '../components/FeedPage/ListingFeed';
+import CreateListing from '../components/CreateListingPage/CreateListing';
 import Footer from '../components/Footer/Footer';
+const listingsArray = require('./mockListings.json');
 
 const GlobalStyles = createGlobalStyle`
```
Comment (NanaDeMesa) [inline src/app/App.js]: Maybe you want to store your GlobalStyles in an extra file to have a better overview

Comment (NanaDeMesa) [reply src/app/App.js]: ? :)

File: src/components/CreateListingPage/CreateListing.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  font-family: 'PT Mono', monospace;
+`;
+
+const StyledButton = styled.button`
+  background-color: #abc38e;
+  border-radius: 11px;
+  padding: 9px;
+  font-size: 18px;
+  font-family: 'PT Mono', monospace;
+  margin-top: 15px;
```
Comment (NanaDeMesa) [inline src/components/CreateListingPage/CreateListing.js]: Tip: Sort your css-props alphabetical or with an other system to have a better overview

File: src/app/App.js (hunk 1 above)
Comment (AntoniaKro) [reply src/app/App.js]: Good Idea :)

File: src/components/CreateListingPage/CreateListing.js (hunk 2 above)
Comment (AntoniaKro) [reply src/components/CreateListingPage/CreateListing.js]: Done

**Pull request merged**